backend/api/tests/test_serializers.py

```diff
@@ -325,13 +325,13 @@ def test_data_complete_false_after_long_time_no_report(self):
         serializer = FactorySerializer(factory)
         assert not serializer.data["data_complete"]
 
-    def test_allow_empty_factory_type(self, request_body):
+    def test_allow_empty_ohshown_event_type(self, request_body):
         request_body.pop('type')
         serializer = FactorySerializer(data=request_body)
 
         assert serializer.is_valid()
 
-    def test_allow_None_factory_type(self, request_body):
+    def test_allow_None_ohshown_event_type(self, request_body):
```

Comment:
Thanks for renaming these function names. In this pull request, some function names are renamed and some of them are not.